client/release/scripts/release.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+# Helper script to prepare a JS Release for the SDKs.
+
+version_sdk_core=$(< client/release/version-sdk-core)
+
+# Extract build and version_sdk number from the configuration.ts
+build=$(awk -F "['\"]" '/SDK_BUILD_NUMBER =/{print $2}' "client/release/version.js" | tr -d '\n')
+version_sdk=$(awk -F "['\"]" '/SDK_VERSION =/{print $2}' "client/release/version.js"| tr -d '\n')
+
+release_notes=$(< client/release/RELEASE-NOTES)
+
+core_modified="${1}"
+
+# Function to execute upon exit
+cleanup() {
+    echo "Performing cleanup tasks..."
+
+    # Revert all the updates to package.json back to main branch versions
+    git checkout main -- wasm/package.json
+    git checkout main -- client/package.json
+    git checkout main -- examples/package.json
+
+    exit 1   
+}
+
+# Set the trap to call the cleanup function on exit
+trap cleanup SIGINT ERR
+
+# Check if Github CLI is installed
+if ! command -v gh &> /dev/null; then
+	echo "gh is not installed";\
+	exit 1;\
+fi
+
+# Ensure GITHUB_TOKEN env var is set
+if [ -z "${GITHUB_TOKEN}" ]; then
+  echo "GITHUB_TOKEN environment variable is not set."
+  exit 1
+fi
+
+if [ "$core_modified" = "true" ]; then
+    cd wasm
+    # Update core version number to the latest
+    npm version "${version_sdk_core}"
+    # Check if all files pertaining to sdk core are included
+    npm publish --dry-run --tag beta
+
+    read -p "Is everything good? (y/n) " files_are_ok
+
+    case "$files_are_ok" in
+        y)
+            # npm publish --tag beta
+            # npm dist-tag add "@1password/sdk-core@$version_sdk_core" latest
+            echo "Publishing and tagging completed."
```

Comment:
```suggestion
            echo "Publishing and tagging on NPM completed."
```
This was a bit confusing as I thought we are tagging GitHub code after core release

Reply:
Done!